index.js

```diff
@@ -1,58 +1,81 @@
-let library = [];
+/* eslint-disable max-classes-per-file */
+class Book {
+  constructor(title, author, id) {
+    this.title = title;
+    this.author = author;
+    this.id = id;
+  }
+}
+
+class Library {
+  constructor() {
+    this.data = [];
+  }
+
+  addBook(book) {
+    this.data.push(book);
+    localStorage.setItem('library', JSON.stringify(this.data));
+    this.updateDisplay(book);
+  }
+
+  removeBook(id) {
+    this.data = this.data.filter((bookObj) => bookObj.id !== id);
+    localStorage.setItem('library', JSON.stringify(this.data));
+    window.location.reload();
+  }
+
+  updateDisplay(bookObj) {
+    const bookList = document.getElementById('book-list');
+    const book = document.createElement('LI');
+    book.innerHTML = `Title: ${bookObj.title} <br> Author: ${bookObj.author}`;
+    const deletetBtn = document.createElement('button');
+    deletetBtn.innerHTML = 'Delete';
+    deletetBtn.addEventListener('click', () => {
+      this.removeBook(bookObj.id);
+    });
+
+    book.appendChild(deletetBtn);
+    bookList.appendChild(book);
+  }
+
+  saver() {
+    this.data.forEach((book) => {
+      this.updateDisplay(book);
+    });
+  }
+}
```

Comment:
- [ ] <b>REQUIRED:</b> To avoid eslint error above, there is no point in creating this class `Library`. you can just use the book class and move all those functions in `Library` class to `Book` class. Here is an example:
```js
class Books {
  constructor() {
    this.books = [];
  }

  addBook(title, author) {
     // code
  }

  removeBook(bookID) {
      // code
  }

  // getter method
  getBooks() {
    // code
  }

```